stored_requests/config/config.go

```diff
@@ -269,32 +225,37 @@ func newHttpEvents(client *http.Client, timeout time.Duration, refreshRate time.
 	return httpEvents.NewHTTPEvents(client, endpoint, ctxProducer, refreshRate)
 }
 
-func newFilesystem(configPath string) stored_requests.AllFetcher {
-	glog.Infof("Loading Stored Requests from filesystem at path %s", configPath)
+func newFilesystem(dataType config.DataType, configPath string) stored_requests.AllFetcher {
+	glog.Infof("Loading Stored %s data from filesystem at path %s", dataType, configPath)
 	fetcher, err := file_fetcher.NewFileFetcher(configPath)
 	if err != nil {
-		glog.Fatalf("Failed to create a FileFetcher: %v", err)
+		glog.Fatalf("Failed to create a %s FileFetcher: %v", dataType, err)
 	}
 	return fetcher
 }
 
-func newPostgresDB(cfg config.PostgresConnection) *sql.DB {
+func newPostgresDB(dataType config.DataType, cfg config.PostgresConnection) *sql.DB {
 	db, err := sql.Open("postgres", cfg.ConnString())
 	if err != nil {
-		glog.Fatalf("Failed to open postgres connection: %v", err)
+		glog.Fatalf("Failed to open %s postgres connection: %v", dataType, err)
 	}
 
 	if err := db.Ping(); err != nil {
-		glog.Fatalf("Failed to ping postgres: %v", err)
+		glog.Fatalf("Failed to ping %s postgres: %v", dataType, err)
 	}
 
 	return db
 }
 
 // consolidate returns a single Fetcher from an array of fetchers of any size.
-func consolidate(fetchers []stored_requests.AllFetcher) stored_requests.AllFetcher {
+func consolidate(dataType config.DataType, fetchers []stored_requests.AllFetcher) stored_requests.AllFetcher {
 	if len(fetchers) == 0 {
-		glog.Warning("No Stored Request support configured. request.imp[i].ext.prebid.storedrequest will be ignored. If you need this, check your app config")
+		switch dataType {
+		case config.RequestDataType:
+			glog.Warning("No Stored Request support configured. request.imp[i].ext.prebid.storedrequest will be ignored. If you need this, check your app config")
+		default:
+			glog.Warningf("No Stored %s support configured. If you need this, check your app config", dataType)
```

Comment:
This should be unreachable in practice, right?

Reply:
It does happen with the default config for video section, and the original error was confusing too as I've shown in #1446 

Original startup messages:

>...
No Stored Request support configured. request.imp[i].ext.prebid.storedrequest will be ignored. If you need this, check your app config
No Stored Request cache configured. The Fetcher backend will be used for all Stored Requests.

After the change:

>...
No Stored Video support configured. If you need this, check your app config
No Stored Video cache configured. The Video Fetcher backend will be used for all data requests

Reply:
Ah, gotcha. Much improved. Thank you.